ckanext/dia/schema.py

```diff
@@ -1,10 +1,11 @@
-from ckan.lib.navl.validators import ignore_missing, not_empty, ignore, not_missing
+from ckan.lib.navl.validators import (
+    ignore_missing, not_empty, ignore, not_missing)
 
 
 def default_extras_schema():
     return {
         'id': [ignore],
-        'key': [not_empty, unicode],
+        'key': [not_empty, str],
```

Comment:
Hmm, not sure about this one

Reply:
maybe should be `six.text_type`??

Reply:
Yes I think this should be. Did `futurize` not pick it up?

Reply:
Nope, will sort this now